bootstrap_config.py

```diff
@@ -72,74 +83,76 @@ def get_all_channels_and_categories(guild_id: int | str) -> tuple[dict[str, str]
     return channels, categories
 
 
-def get_webhook(webhook_id_: int) -> dict:
-    """Fetches a particular webhook by its id."""
-    webhooks_url = f"{BASE_URL}/webhooks/{webhook_id_}"
-    response = requests.get(url=webhooks_url, headers=HEADERS)
-    if response.status_code == 200:
-        return response.json()
-
-    return {}
+def webhook_exists(webhook_id_: int, client: DiscordClient) -> bool:
+    """A predicate that indicates whether a webhook exists already or not."""
+    try:
+        client.get(f"webhooks/{webhook_id_}")
+        return True
+    except HTTPStatusError:
+        return False
 
 
-def create_webhook(name: str, channel_id_: int) -> str:
+def create_webhook(name: str, channel_id_: int, client: DiscordClient) -> str:
     """Creates a new webhook for a particular channel."""
-    create_webhook_url = f"{BASE_URL}/channels/{channel_id_}/webhooks"
     payload = {"name": name}
 
-    response = requests.post(url=create_webhook_url, headers=HEADERS, json=payload)
+    response = client.post(f"channels/{channel_id_}/webhooks", json=payload)
     new_webhook = response.json()
     return new_webhook["id"]
 
 
-config_str = "#Roles\n"
-
-all_roles = get_all_roles(guild_id=GUILD_ID)
-
-for role_name in _Roles.__fields__:
-
-    role_id = all_roles.get(role_name, None)
-    if not role_id:
-        log.warning(f"Couldn't find the role {role_name} in the guild, PyDis' default values will be used.")
-        continue
+if __name__ == '__main__':
```

Comment:
I don't think we should have this guard. This file should in theory not be imported anywhere, and it adds an unnecessary layer of indentation.

Reply:
removed in 469d609ce53a3e7e1378672c4787cc80444450ba